### DISCUSSION TITLE
Tipe Data Naive Bayes

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#460] [Q&A] [answered]
Selamat siang
Saya ingin memastikan kembali
Apakah naive bayes bisa untuk tipe data numerik?
Dan kalau bisa penulisan pada saat predict menggunakan tanda petik atau tidak?

diva <- data.frame(Age = 23, 
                    Gender = "Female",
                    Occupation = "Student")

atau 

diva <- data.frame(Age = ''23", 
                    Gender = "Female",
                    Occupation = "Student")

Terima kasih

**Comment by VictorNugraha** [comment, marked as answer]
Dear @ageljonathan,

Model Naive Bayes bisa diperuntuhkan untuk data numerik maupun kategorikal. 

Untuk pertanyaan kedua, jika Bapak ingin melakukan prediksi pada variabel numerik tidak usah diberikan tanda petik.

Berikut yang dapat saya sampaikan, semoga bermanfaat dan selamat mencoba. 

Terima kasih.

**Comment by ageljonathan** [comment]
Baik mas
Thank you
